app/src/main/java/io/appium/uiautomator2/model/api/gestures/RectModel.java

```diff
@@ -0,0 +1,30 @@
+package io.appium.uiautomator2.model.api.gestures;
+
+import android.graphics.Rect;
+
+import io.appium.uiautomator2.model.RequiredField;
+import io.appium.uiautomator2.model.api.BaseModel;
+
+public class RectModel extends BaseModel {
+    @RequiredField
+    public Double top;
+    @RequiredField
+    public Double left;
+    @RequiredField
+    public Double width;
+    @RequiredField
+    public Double height;
+
+    public RectModel() {}
```

Comment:
Codacy found an issue: [Avoid unnecessary constructors - the compiler will generate these for you](https://app.codacy.com/manual/dpgraham/appium-uiautomator2-server/pullRequest?prid=6222243)